### ISSUE TITLE
testing CP

### ISSUE THREAD

**Issue by ddalthorp** [ddalthorp/GenEst#137]
Using Jeff's CP_trials.csv data...

Wow. The analysis takes several minutes on my computer. 

- [x] I see lots of nice AIC's but I can only get a blank figure

- [x] is there any way to put AIC's on the drop down menus for model selection?

- [x] we need table(s) of parameter estimates. What goes in them?

**Comment by DAPPERstats** [comment]
yeah it's definitely slower than the previous version and than the SE models, but it's to be expected given the number of models involved now and the first-cut attempts at the optimization procedures. totally room for improvement on that end of things

ooo good catch with those figures. the code needed a patch to deal with Inf values (I had only dealt with NAs) when sizing the figures. i've added a patch and will post it shortly.

there are model parameter tables just like on the SE tab, under the model tables tab:
![image](https://user-images.githubusercontent.com/23511502/39469199-ebcbf6a2-4ceb-11e8-86fd-648ae38bb49c.png)

we could add AIC to the model drop down but i do worry about that being really cluttered, especially with multiple predictor values and especially for the CP models that also include the distribution names
the models in the dropdown are automatically sorted by AICc, so the best model is chosen already.

**Comment by ddalthorp** [comment]
- [ ] for predictor variables in GUI, do not allow "Size" (if selected in General Inputs) or left or right intervals.

**Comment by ddalthorp** [comment]
figs still don't work for me in GUI.

Using Jeff's CP_trials.csv data with best AICc model (Weibull, l ~ Visibility, s ~ 1) and small carcasses

```
Listening on http://127.0.0.1:5580
Warning: Error in plot.window: need finite 'xlim' values
Stack trace (innermost first):
    111: plot.window
    110: localWindow
    109: plot.default
    108: plot
    107: plot.survfit
    106: plot
    105: cpmSetSpecCPCellPlot
    104: plot.cpmSet
    103: plot
    102: renderPlot
     92: <reactive:plotObj>
     81: plotObj
     80: origRenderFunc
     79: output$fig_CP
      4: <Anonymous>
      3: do.call
      2: print.shiny.appobj
      1: <Promise>

```

**Comment by ddalthorp** [comment]
"we could add AIC to the model drop down..."

If we can make the drop down wider, keep the model left-justified and right-justify the delta AICc, it would be useful information to include.

**Comment by DAPPERstats** [comment]
I've included the AICcs in the model dropdowns now as of v 0.0.5.9 (#163 )
I tried to make the labels work as you suggested, but something in the R -> HTML transfer ends up lopping off extra spaces, so there's no simple way to do the justification like that. 
What I did here instead is add the AICc values parenthetically after the model name, and then still make the dropdown width flexible so it scales with the model name width. 
The parentheses help keep it clean while still providing the additional info.
